openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -1165,6 +1168,48 @@ def revert_changes(library_key):
     )
 
 
+# V1/V2 Compatibility Helpers
+# (Should be removed as part of
+#  https://github.com/openedx/edx-platform/issues/32457)
+# ======================================================
+
+def get_v1_or_v2_library(
```

Comment:
So I understood the library_tools interface as being, for now, the interface between the xblock itself and the v1 and v2 library code. Is the merit of this having 1 less api to import in the tasks file, as we have to pull in the libraries again there? Should we refactor the library_tools api to use this method instead of get_library?